ibm/service/database/resource_ibm_database.go

```diff
@@ -2837,6 +2870,39 @@ func validateGroupHostFlavor(groupId string, resourceName string, group *Group)
 	return nil
 }
 
+func validateMultitenantMemoryCpu(groupId string, resourceName string, resourceDefaults *Group, group *Group) error {
+	// TODO: Replace this with resourceDefaults.Memory.CPUEnforcementRatioCeiling when it is fixed
+	cpuEnforcementRatioCeiling := 16384
+	cpuEnforcmentRationMb := resourceDefaults.Memory.CPUEnforcementRatioMb
+
+	// This means the CPUEnforcementRatioMb was not sent, which means we are dealing with a non-multitenant going to a multitenat
+	if cpuEnforcmentRationMb == 0 {
+		cpuEnforcmentRationMb = 8192
+	}
```

Comment:
if `cpuEnforcmentRationMb` is coming from `resourceDefaults` is this logic required here?

Reply:
@obai-1 There is an edge case, when a user is has an existing databases and is going from `non-multitenant` to `multitenant`. In that case, we use the `deployments/:id/groups` API endpoint to get the database current configuration.

However, the `deployments/:id/groups` does not have the `host_flavor` query param. In other words, we do not get the `cpuEnforcementRatioCeiling` or the `cpuEnforcementRatioMb`.

So we have one of 2 options:
1. Call the `deployables/{type}/group`. Process the result, extract the ratios from it, and pass those into the function as well. This is a large amount of code that will have to be removed regardless as the enforcement ratios and query parameter are going to go away soonish, per the following "when the old hosting model goes away, so do these query string parameters".
2. Account for the edge case by hard-coding the cpuEnforcementRatio in the event the API didn't return it.

Reply:
> as the enforcement ratios and query parameter are going to go away soonish

Hmm.. unless I am missing something, they are not going away anytime soon..

Reply:
Also hardcoding these numbers here is not safe. Those numbers could change down the line, and could also be db specific. Remember once we release it out to the wild, it is hard to take it back :)